insitu/templates/about.html

```diff
@@ -6,12 +6,34 @@
 {% block content %}
   <h1>About</h1>
 
+  <h3><b>&emsp;Sentry errors</b></h3>
+  <br>
+  <table class="sentry-table">
```

Comment:
Use https://getbootstrap.com/docs/4.0/content/tables/#striped-rows from bootstrap for table styling

add thead, tbody tags